### ISSUE TITLE
Support for MSpec 

### ISSUE THREAD

**Issue by Exef** [CodeConnect/AliveFeedback#6]
Is there any possibility to support Machine.Specification (https://github.com/machine/machine.specifications) as next test framework?

**Comment by JoshVarty** [comment]
We'll definitely look into it. Are there any peculiarities to this framework when compared to MSTest or XUnit?

**Comment by Exef** [comment]
Yes, the syntax is based on RSpec. It's rather BDD than TDD framework.
